### PR TITLE
Initial implementation of museums

### PR DESCRIPTION
**Pull request by lennym**
Has a bunch of address things left pending because they're reliant on other work happening in other PRs to refactor address loops and custom options. Will revisit once addresses are in master.

### CODE REVIEW

File: apps/museums/index.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    },
+    '/contact-address-postcode': {
+      next: '/contact-address-select'
+    },
+    '/contact-address-select': {
+      next: '/confirm'
+    },
+    '/confirm': {
+      template: 'confirm',
+      controller: require('./controllers/confirm'),
```
Comment (JoeChapman) [inline apps/museums/index.js]: use https://github.com/UKHomeOfficeForms/hof-controllers/blob/master/lib/confirm-controller.js intstead

File: apps/museums/translations/src/en/pages.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "header": "Is the contact address the same as an address previously given?"
+  },
+  "contact-address-postcode": {
+    "header": "Placeholder pending address lookup refactor"
+  },
+  "contact-address-select": {
+    "header": "Placeholder pending address lookup refactor"
+  },
+  "confirmation": {
+    "next": "We will review your application. We’ll instruct the local police force to make enquiries on behalf of the Secretary of State about matters related to this application. In this case, you should hear from the police within the next X weeks. If insufficient information has been provided for the Home Office to assess this application, we will be in touch with you within the next X days/weeks.",
```
Comment (JoeChapman) [inline apps/museums/translations/src/en/pages.json]: "... the next X days/weeks." Should there be a locals placeholder here?

Comment (lennym) [inline apps/museums/translations/src/en/pages.json]: Probably, the copy is taken straight from the prototype.

Comment (JoeChapman) [inline apps/museums/translations/src/en/pages.json]: Np, apparently there is a story for this.

**Comment by JoeChapman** [comment]
When tests fixed, LGTM

**Comment by lennym** [comment]
@sulthan-ahmed Any particular reason you have added a template for confirmation page which is identical to the common one?

**Comment by sulthan-ahmed** [comment]
Good spot, thanks.  I've now deleted the specific confirmation.html for museums and just using the generic one in common @lennym

**Pull request merged**